File: remocra/src/main/webapp/js/app/remocra/model/OldebLocataire.js
Ext.define('Sdis.Remocra.model.OldebLocataire', {
    extend: 'Ext.data.Model',

    fields: [{
        name: 'id',
        type: 'int',
        useNull: true
    }, {
        name: 'organisme',
        type: 'boolean'
    }, {
        name: 'raisonSociale',
        type: 'string'
    }, {
        name: 'civilite',
        type: 'string'
    }, {
        name: 'nom',
        type: 'string'
    }, {
        name: 'prenom',
        type: 'string'
    }, {
        name: 'telephone',
        type: 'string'
    }, {
        name: 'email',
        type: 'string'
    }, {
        name: 'oldeb',
        type: 'fk',
        useNull: true
    }

    ],
    proxy: {
        type: 'remocra.rest',
        url: Sdis.Remocra.util.Util.withBaseUrl('../locataire')
    }
});

File: remocra/src/main/java/fr/sdis83/remocra/domain/remocra/OldebLocataire.java
package fr.sdis83.remocra.domain.remocra;

import javax.persistence.Column;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.validation.constraints.NotNull;
import org.springframework.roo.addon.javabean.RooJavaBean;
import org.springframework.roo.addon.jpa.activerecord.RooJpaActiveRecord;
import org.springframework.roo.addon.tostring.RooToString;

@RooJavaBean
@RooJpaActiveRecord(versionField = "", table = "oldeb_locataire", schema = "remocra")
@RooToString
public class OldebLocataire {

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "id")
  private Long id;

  @Column(name = "organisme")
  private Boolean organisme;

  @Column(name = "raison_sociale")
  private String raisonSociale;

  @Column(name = "civilite")
  private String civilite;

  @Column(name = "nom")
  @NotNull
  private String nom;

  @Column(name = "prenom")
  private String prenom;

  @Column(name = "telephone")
  private String telephone;

  @Column(name = "email")
  private String email;

  @ManyToOne
  @JoinColumn(name = "oldeb", referencedColumnName = "id")
  private Oldeb oldeb;
}
